components/list/test/list.vdiff.js

```diff
@@ -541,12 +541,11 @@ describe('list', () => {
 
 		[
 			{ name: 'default', template: createDraggableList() },
-			{ name: 'focus', template: createDraggableList(), action: elem => focusElem(elem.querySelector('[key="1"]')) },
-			{ name: 'hover', template: createDraggableList(), action: elem => hoverElem(elem.querySelector('[key="1"]')) },
```

Comment:
This test scenario was duplicated in `hover list item`. Let me know if there was a reason for both. I kept `hover list item` and then renamed `focus` to `focus list item` to align.